voter-dapp/src/App.js

```diff
@@ -1,24 +1,47 @@
 import React from "react";
+import { MuiThemeProvider, createMuiTheme } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
 import { drizzleReactHooks } from "drizzle-react";
 import "./App.css";
+import Header from "./Header.js";
 import ActiveRequests from "./ActiveRequests.js";
-import ResolvedRequests from "./ResolvedRequests.js";
+import { red, grey } from "@material-ui/core/colors";
+import AppBar from "@material-ui/core/AppBar";
 
 function App() {
+  const theme = createMuiTheme({
+    palette: {
+      primary: red, //TODO --> find actual UMA red
+      secondary: grey
```

Comment:
nit: there should be a list of secondary colors in the press kit too. See the [sponsor dapp styling](https://github.com/UMAprotocol/protocol/blob/a4b6c69bc521e554493af4f330e04af1def20f21/sponsor-dapp/src/App.js#L32-L57) for how we did this there. I only show that example to illustrate how to do this - don't feel obligated to match the look and feel of the sponsor dapp.